website/views/index.jade

```diff
@@ -4,11 +4,13 @@ html(ng-app="habitrpg", ng-controller="RootCtrl", ng-class='{"applying-action":a
   head
     title=env.t('titleIndex')
     // ?v=1 needed to force refresh
-    link(rel='shortcut icon', href='#{env.getBuildUrl("favicon.ico")}?v=4')
+    link(rel='shortcut icon', sizes="32x32", href='#{env.getBuildUrl("favicon.ico")}?v=4')
+    link(rel='shortcut icon', sizes="144x144", href='#{env.getBuildUrl("favicon_192x192.png")}?v=4')
 
     meta(charset='utf-8')
     meta(name='viewport', content='width=device-width, initial-scale=1.0')
     meta(name='apple-mobile-web-app-capable', content='yes')
+    meta(name='mobile-web-app-capable', content='yes')
```

Comment:
BTW, when I add it to my homescreen, there is no icon displayed using iOS9, iPhone6. Maybe I should find out why and fix it in this PR.
> On 19 Oct 2015, at 20:00, negue <notifications@github.com> wrote:
> 
> In website/views/index.jade:
> 
> >  
> >      meta(charset='utf-8')
> >      meta(name='viewport', content='width=device-width, initial-scale=1.0')
> >      meta(name='apple-mobile-web-app-capable', content='yes')
> > +    meta(name='mobile-web-app-capable', content='yes')
> with that it should be able to pin habitica as "web app" to the android launcher , right?
> 
> Since we already have the ios version on it, we also could add this ....
> 
> ping @veeeeeee @lemoness
> 
> —
> Reply to this email directly or view it on GitHub.
>